lib/Service/ConfigService.php

```diff
@@ -329,16 +331,43 @@ public function contactsBackendType(): int {
 	 * @return bool
 	 * @deprecated
 	 * should the password for a mail share be send to the recipient
-	 *
 	 */
-	public function sendPasswordByMail() {
-		if ($this->isContactsBackend()) {
-			return false;
+	public function sendPasswordByMail(): bool {
+		return false;
```

Comment:
A config function that *always* returns `false`? What's the purpose of this? If it's just a leftover from the past, why not remove it altogether? This looks suspicious :thinking:

Reply:
well, it is marked as deprecated ;-)